1.8/development/dcos-integration.md

```diff
@@ -12,17 +12,28 @@ When a DC/OS Service is installed and ran on DC/OS, the service is generally dep
 
 Admin Router currently supports only one reverse proxy destination.
 
-In order to allow Admin Router to function as a reverse proxy for the DC/OS Service one of the following methods should be used
+## Service Endpoints
 
-## Framework Web UI URL
+Admin Router allows marathon tasks to define custom service UI and HTTP endpoints, which are made available as `/services/<service-name>`. This can be achieved by setting the following marathon task labels:
 
-If the DC/OS Service will register with Mesos as a framework, the `webui_url` framework property may be specified and will be used by Admin Router to proxy requests to the service.
+```
+"labels": {
+    "DCOS_SERVICE_NAME": "service-name",
```

Comment:
If we say `<service-name>` (literal) in the text above, then we should also use `<service-name>"` here.